app/mobile_app/mobile_app_registry.py

```diff
@@ -1,35 +1,27 @@
 from typing import List
-from flask import current_app
-from .mobile_app_types import MobileAppType
+
 from .mobile_app import MobileApp
+from .mobile_app_types import MobileAppType
 
 
 class MobileAppRegistry:
-    _instance = None
-
-    def __new__(
-        cls,
-        *args,
-        **kwargs,
-    ):
-        if not cls._instance:
-            cls._instance = super().__new__(cls)
-        return cls._instance
-
-    def __init__(self):
+    def __init__(self, logger):
+        self.logger = logger
         self._registry = {}
+
+        logger.info('Intializing MobileAppRegistry')
```

Comment:
@David This I included temporarily to test on DEV to test this was only initialized once.  I planned on removing this log.

Reply:
I would leave it.  It should only be called once.